src/util/Rewards.cpp

```diff
@@ -102,28 +102,37 @@ namespace YimMenu::Rewards
 					STREAMING::REQUEST_MODEL(hash, false);
 					ScriptMgr::Yield();
 				}
+	
 				Vector3 coords = ENTITY::GET_ENTITY_COORDS(PLAYER::PLAYER_PED_ID(), true, false);
-				Object obj = OBJECT::CREATE_OBJECT(hash, coords.x, coords.y, coords.z, true, NETWORK::NETWORK_IS_HOST_OF_THIS_SCRIPT(), 1, 0, 1);
+				float forwardX = ENTITY::GET_ENTITY_FORWARD_X(PLAYER::GET_PLAYER_PED_SCRIPT_INDEX(PLAYER::PLAYER_ID()));
```

Comment:
You can use `GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS` and set the Y coordinate to be 2 paces in front.

Reply:
Either/or will work.  Ive never used that native before so i did not know about this one lol

Reply:
Fixed this, ready for review of the rest and merge!